include/picongpu/fields/incidentField/Solver.hpp

```diff
@@ -213,6 +216,52 @@ namespace picongpu
                     auto const& incidentField = *dc.get<T_IncidentField>(T_IncidentField::getName(), true);
                     using Functor
                         = UpdateFunctor<decltype(dataBox), T_CurlIncidentField, T_FunctorIncidentField, T_axis>;
+                    auto functor = Functor{
+                        parameters.sourceTimeIteration,
+                        parameters.direction,
+                        curlCoefficient,
+                        incidentField.getUnit()};
+                    functor.updatedField = dataBox;
+                    functor.isUpdatedFieldTotal = isUpdatedFieldTotal;
+
+                    // Convert to the local domain indices
+                    auto const& subGrid = Environment<simDim>::get().SubGrid();
+
+                    // Set local domain information
+                    auto& gridController = pmacc::Environment<simDim>::get().GridController();
+                    auto const localDomainIdx = gridController.getPosition();
+                    auto const numLocalDomains = gridController.getGpuNodes();
+
+                    const DataSpace<DIM3> periodic
+                        = Environment<simDim>::get().EnvironmentController().getCommunicator().getPeriodic();
+
+                    for(uint32_t d = 0u; d < DIM3; ++d)
+                    {
+                        bool isLaserAxisDirection = d == T_axis;
+
+                        // constexpr bool extendTransversalHuygensSurface
+                        //     =
+                        //     profiles::MakePeriodicTransversalHuygensSurfaceContiguous<T_FunctorIncidentField>::value;

```

Comment:
What is up with this dead/commented code?

Reply:
deleted it, missed to delete it, dead code is always bad and must be removed.